core/pom.xml

```diff
@@ -52,8 +52,8 @@
             <artifactId>apache-mime4j-core</artifactId>
         </dependency>
         <dependency>
-            <groupId>commons-logging</groupId>
-            <artifactId>commons-logging</artifactId>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
```

Comment:
And what if we remove that dependency?

(Only use raw SLF4J APIs everywhere and not common logging ones?)